docs/sources/reference/run.md

```diff
@@ -1122,8 +1122,10 @@ container's `/etc/hosts` entry will be automatically updated.
 
 ## VOLUME (shared filesystems)
 
-    -v=[]: Create a bind mount with: [host-dir]:[container-dir]:[rw|ro].
-           If "container-dir" is missing, then docker creates a new volume.
+    -v=[]: Create a bind mount with: [host-dir:]container-dir[:rw|ro].
```

Comment:
Tics not quotes please.  Sorry, I see this was let through before and you were just following a bad precedent.